utt/privacy-wallet-service/proto/api/v1/wallet-api.proto

```diff
@@ -112,6 +129,8 @@ message PrivacyWalletRequest {
         GenerateBurnTx generate_burn_tx_request = 7;
         GenerateTransferTx generate_transfer_tx_request = 8;
         GetStateRequest get_state_request = 9;
+        SetAppDataRequest set_app_data_request = 11;
```

Comment:
set tag to 10

Reply:
10 is already used by `google.protobuf.Duration timeout = 10;`

Reply:
ah didn't notice it. We could move the timeout to tag 1 and keep others consecutive to be more clear.